products/opensuse/main.pm

```diff
@@ -329,6 +329,7 @@ sub load_reboot_tests {
             }
         }
         loadtest "installation/first_boot";
+        loadtest "locale/keymap_or_locale";
```

Comment:
I recommend to schedule it as part of the console tests. That should be still after `first_boot` but not called in the context of INSTALLONLY-jobs, e.g. as used for image publication where we only want to conduct the installation, check that the system boots, and then publish the image to be used in follow-up tests

Reply:
Works fine for SLE, however not for opensuse, due to there are update tests scheduled between first_boot and console tests. As long as there is an specific problem with INSTALLONLY-jobs, maybe we can use 'loadtest "locale/keymap_or_locale" unless get_var("INSTALLONLY");'. Wdyt ?

Reply:
hm, I guess what we are trying to do here is to define another class of test modules that should go between first boot and potential update tests. This is why I suggested in https://progress.opensuse.org/issues/30586 to select a tty that is so far unused and check the getty on that one so that we do not need to care if we schedule the test before or after any other console based test. IMHO there should be no problem about the exact schedule location then and we should just schedule this test module as part of the other console tests. Can you try that?

Reply:
Keep in mind that we refactored the schedule in the main.pm and lib/main_common.pm files in the meantime hence the conflict in your PR. I assume now your test schedule should go into lib/main_common.pm in a openSUSE/SLE common schedule part anyway